exporter/redis.go

```diff
@@ -659,6 +659,8 @@ var errNotFound = errors.New("key not found")
 // getKeyInfo takes a key and returns the type, and the size or length of the value stored at that key.
 func getKeyInfo(c redis.Conn, key string) (info keyInfo, err error) {
 
+	var size int64
```

Comment:
What's the reason for pulling this out?

Reply:
This is reported by `vetshadow`, a check for shadowed variables. The problem is, as the commit message said, the `err` variable shadowing. 

As I said, if you do not agree, I can drop it.